core/domain/platform_parameter_list.py

```diff
@@ -35,6 +35,7 @@ class ParamNames(enum.Enum):
     END_CHAPTER_CELEBRATION = 'end_chapter_celebration'
     CHECKPOINT_CELEBRATION = 'checkpoint_celebration'
     ANDROID_BETA_LANDING_PAGE = 'android_beta_landing_page'
+    BLOG_PROJECT = 'blog_project'
```

Comment:
'project' is a bit redundant (everything can be considered a 'project'), I suggest striking that.

As for 'blog', I think that's fine name-wise (though it is rather generic, but it seems the blog itself is actually pretty generic).

Reply:
@Rijuta-s Could you please use a name that conveys exactly what the flag is controlling? I can't tell. Maybe in the description you can list the specific pages that are access-controlled by the flag.

Also could you please show a video with the flag turned on and off? When the flag is turned off any links to the blog page from other pages should also be removed.

Reply:
Changed to `BLOG_PAGES` and added a description-   'This flag is for blog home page, blog author profile page and blog post page.'.